operations/flags.go

```diff
@@ -158,6 +158,13 @@ func addSyncStatusesFlag(flags ...cli.Flag) []cli.Flag {
 	})
 }
 
+func addJSONOutputFlag(flags ...cli.Flag) []cli.Flag {
+	return append(flags, cli.BoolFlag{
+		Name:  joinFlagNames(jsonFlagName, "j"),
+		Usage: "output JSON instead of text",
```

Comment:
I think it would be good if the patch version of this flag mentioned that warnings are suppressed and that it has to be used with -y, so we may not want to have a variable for this flag

Reply:
That makes sense, I'll switch it to be standalone and include that